workflows/speciesabundance.nf

```diff
@@ -68,19 +68,39 @@ workflow SpAnce {
 
     ch_versions = Channel.empty()
 
+    // Track processed IDs
+    def processedIDs = [] as Set
+
     // Create a new channel of metadata from a sample sheet
     // NB: `input` corresponds to `params.input` and associated sample sheet schema
     input = Channel.fromSamplesheet("input")
+        .map { meta, fastq_1, fastq_2 ->
+            // Set meta.id to meta.irida_if if sample_name is not provided in the samplesheet
```

Comment:
In comment `irida_if` should be `irida_id`.

Reply:
Typo fixed here: [3a081bb](https://github.com/phac-nml/speciesabundance/pull/24/commits/3a081bb0c6174f9c1e62e5ef94c8f8ce89b5c897)